### DISCUSSION TITLE
I want to add the line break in the blockquote

### DISCUSSION THREAD

**Discussion by no13bus** [gonzalezreal/swift-markdown-ui#290] [Q&A] [answered]
for example, if the markdown is like this:

> No Man Is An Island
> entire of itself;
> every man is a piece of the continent,
> a part of the main;

I want it looks like the github style. 
<img width="384" alt="Screenshot 2024-01-05 at 22 58 06" src="https://github.com/gonzalezreal/swift-markdown-ui/assets/3690806/d2783f45-8190-4bf9-8344-fc3bac5f52f2">

but now in the app, it just one line.
<img width="356" alt="Screenshot 2024-01-05 at 22 59 08" src="https://github.com/gonzalezreal/swift-markdown-ui/assets/3690806/8d994151-7b4e-4ca3-8aa1-0f1ae665ac15">

**Comment by gonzalezreal** [comment, marked as answer]
As stated in the [GitHub Flavored Markdown Spec](https://github.github.com/gfm/#hard-line-breaks), you can add two or more spaces before the line break to render a hard line break.

Another option is to insert an HTML line break element at the end of each line:

```markdown
> No Man Is An Island<br />
> entire of itself;<br />
> every man is a piece of the continent,<br />
> a part of the main;
```

**Reply by no13bus** [reply]
is it possible to use other markdown spec to avoid this?

**Reply by gonzalezreal** [reply]
At the moment, there is no other way than being explicit about the hard line breaks, according to the spec.
